src/inspstring.cpp

```diff
@@ -123,9 +123,9 @@ std::string Base64::Encode(const void* data, size_t length, const char* table, c
 	for (size_t idx = 0; idx < length; )
 	{
 		// Base64 encodes three octets into four characters.
-		uint32_t octet1 = idx < length ? udata[idx++] : 0;
```

Comment:
This seems like a false positive to me? `idx++` may increase udata past the end of the array.